torch/onnx/_internal/exporter.py

```diff
@@ -544,13 +431,15 @@ def _export_fx_to_onnx(
             )
             # ONNX does not support None inputs. During graph building, all None inputs
             # are removed. Here we register this step to input adapter.
-            self.adapt_input(fx_exporter.RemoveNoneInputStep, fx_module_args, {})
+            self.input_adapter.append_step(io_adapter.RemoveNoneInputStep())
```

Comment:
I'm ok about this for now. But as you said once pass start to interact with adapter, `fx_module_args` will be involved.

Reply:
Discussed offline. TL;DR Although today there is no actual need for inputs to be used by passes, hypothetically it might.
We probably will need to refactor if that cases ever happens, but assuming a simpler scenario without such complication for now